### PR TITLE
Add jupyterhub grafana dashboard resource

### PR DESCRIPTION
**Pull request by hemajv**
Adding `GrafanaDashboard` resource to include the Jupyterhub dashboard created at: https://github.com/operate-first/SRE/pull/17.

### CODE REVIEW

File: odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml
Comment (4n4nd) [inline odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml:9]: let's wait for this link to work before we merge?

File: odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml
Comment (4n4nd) [inline odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml:1]: Do we need to add this file to some kustomization.yaml?

Comment (anishasthana) [reply odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml:1]: I think we don't need to as the general overrides folder is already referred to in the kfdef file.

File: odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml
Comment (HumairAK) [inline odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml:4]: hrmm I don't think we need a `-dashboard` suffix, since it's already of `kind: GrafanaDashboard`

File: odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml
Comment (hemajv) [reply odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml:9]: link is available now

File: odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml
Comment (hemajv) [reply odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml]: fixed :+1:

**Review by HumairAK — APPROVED** [review]

**Review by 4n4nd — APPROVED** [review]
LGTM :+1: 
/unhold since the dashboard link works now

**Pull request merged**
